src/components/HelpText/styled/index.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+export const StyledButton = styled.button`
+    border: none;
+    background: transparent;
+
+    &:focus,
+    &:active {
+        outline: 0;
+    }
+`;
+export const StyledTooltip = attachThemeAttrs(styled.div)`
+    width: 244px;
```

Comment:
we should not add a fixed width, the user should control the width of the content they want to show, instead maybe we can use min-width